ticket-monster.adoc

```diff
@@ -20,7 +20,7 @@ pass:[<span><i class="fa fa-desktop fa-9x fa-fw"></i></span>]
 
 Walk through the tutorial to see what Ticket Monster will teach you.
 
-pass:[<span><i class="fa fa-book fa-9x fa-fw"></i></span>]
+pass:[<a href="/ticket-monster/whatisticketmonster"><i class="fa fa-book fa-9x fa-fw"></i></a>]
```

Comment:
This link doesn't work in staging. Take a look at how other similar links are handled. E.g:

        a(href="#{site.base_url}/video/vimeo/92792413")